cibyl/sources/elasticsearch/api.py

```diff
@@ -185,17 +186,29 @@ def get_deployment(self, **kwargs):
             query=query_body
         )
 
+        IP_REGEX = re.compile("ipv(.)")
```

Comment:
This regex pattern is also used in the jenkins source. I think it would be good to move them both out in the utils/filtering.py module for example

Reply:
You're right. I'll change all the REGEX constants to that file and import in both sources.